### PR TITLE
 "IT'S HUUGE, TRUST ME!"

### PR DESCRIPTION
**Pull request by Austinh100**
### Improvements to installation, README update, fixes, and surprise Trump!

- Updates makefile to auto resolve dependencies, should help issue #9 and #6
- Renames generated binaries to be uniform
- Adds some Trump audio
- Adds .dca to gulpfile test line
- Fix for .wav include for airhorn_default

### CODE REVIEW

**Comment by viisual** [comment]
Nice, especially the readme and highly topical sound bytes @Austinh100.

Which audio file format is preferred for the bot?

File: cmd/airhornbot/bot.go [hunk 1]
```diff
@@ -169,12 +169,27 @@ var COW *SoundCollection = &SoundCollection{
...
+	Prefix: "trump",
+	Commands: []string{
+		"!trump",
+	},
+	Sounds: []*Sound{
+		createSound("mess", 10, 250),
+		createSound("bing", 3, 250),
+		createSound("getitout", 1, 250),
+		createSound("tractor", 3, 250),
+		createSound("worstpres", 3, 250),
```
Comment (iopred) [inline cmd/airhornbot/bot.go]: Where is:
createSound("buildawall") ?

File: static/src/Constants.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default {
   },
 
   Audio: {
-    AIRHORN: require('../../audio/airhorn_default.wav')
+    AIRHORN: require('../../audio/airhorn_default.dca')
```
Comment (iopred) [inline static/src/Constants.js]: You don't want to delete the wav file as it is used by the oauth website

File: cmd/airhornbot/bot.go (hunk 1 above)
Comment (Austinh100) [inline cmd/airhornbot/bot.go]: Honestly, I looked but couldn't find a good clip. Find me one and i'll add it :)

File: static/src/Constants.js (hunk 2 above)
Comment (Austinh100) [inline static/src/Constants.js]: Your right, just re-added this.

**Comment by Austinh100** [comment]
Should be good to go now

File: cmd/airhornbot/bot.go (hunk 1 above)
Comment (iopred) [inline cmd/airhornbot/bot.go]: http://trumplings.com/sound/build-wall.mp3

**Comment by mantis8085** [comment]
When is this getting added? Love it

**Comment by b1naryth1ef** [comment]
Hey Austin,

Sorry for the delay responding! Your code all looks fine at this point, but we're of the opinion that airhornbot doesn't believe in taking a political stance. Feel free to use this and deploy your own version of airhornbot (trumpbot!?) for everyone to add/use. Regarding the other changes, I'll try to get some of them included as I spend a bit of time cleaning up over the next few days.

Thanks again!

**Comment by viisual** [comment]
@Austinh100 That was a good write up but I still had a bit of work/hacking to do based on that readme.

Off the top of my head:
Go 1.5, 1.6 required not 1.4 as per that .v3 redis plugin.
Adding the bot to the server isn't as easy as running the webserver by default, or at least, for me it wasn't working.  Seemed like React or Webpack was missing outright.  Either way I went to http://airhorn.solutions and opened the popup, copied the URI and changed the clientID to add the bot to my server.

cheers.

**Pull request closed without merging**